README.md

```diff
@@ -18,17 +18,22 @@ Mapping to other taxonomies
 
 ## Specification Working Group
 
-* Adomni
-* Broadsign
-* Place Exchange
-* The Trade Desk
-* Verizon Media
-* VIOOH
-* Vistar Media
+* Aura Koljonen (Broadsign)
+* Bruno Guerrero (Hivestack)
+* [Christian Collins](https://github.com/christiancollins11) (Vistar)
+* Connor Peltz (Broadsign)
+* [Jasleen Kaur](https://github.com/jasleenk-viooh) (VIOOH)
+* [Jason Shao](https://github.com/jayshao) (Place Exchange)
+* Joshua Berg (MediaMath)
+* [M. Wasfi](https://github.com/mowasfi7) (Verizon)
+* Matthew Mercuri (Samsung)
+* [Nicholas Babb](https://github.com/ndbabb) (Adomni)
+* Sweta Vasudevan (Place Exchange)
+* William Hartley-Booth (The Trade Desk)
 
 ## Adoption
 
-| Participant | Status | Version | Notes |
+| Platform | Status | Version | Notes |
```

Comment:
Hivestack is now supporting the taxonomy 1.0, can we add our name to the list

Reply:
@bruno8g when did you guys start supporting it?